.github/workflows/vs2019/build-cbmc.bat

```diff
@@ -16,7 +16,7 @@ echo "Remove strawberry"
 del /s /q "c:/Strawberry/c/bin"
 
 echo "Configure CBMC with cmake"
-cmake -S. -Bbuild -GNinja
+cmake -S. -Bbuild
```

Comment:
... which is rectified here, these two things should be one commit, with an explanation of what's going on.